lib/blacklight/abstract_repository.rb

```diff
@@ -35,6 +35,12 @@ def find(_id, _params = {})
       raise NotImplementedError
     end
 
+    # Find multiple documents by their ids
+    # @param [Hash] _params query parameters
+    def find_all(params)
```

Comment:
`find_all` might not be the best name as it's only a subset we're actually finding, right?

Reply:
Got a better suggestion? I was inspired by ActiveRecord and Enumerable that use(d) `find_all` to mean "find all the match some condition"

Reply:
is `find_many` confusing? I know we are using it at a different level of abstraction, but it seems to accurately describe it's function.

Reply:
🤷‍♂️ sure